### PR TITLE
update libraries

### PR DESCRIPTION
**Pull request by davidecorreu**

### CODE REVIEW

File: packages/vscode-extension/package.json [hunk 1]
```diff
@@ -139,12 +139,12 @@
 				"command": "suitecloud.updateobject",
 				"key": "ctrl+alt+u",
 				"mac": "cmd+alt+u",
-				"when": "resource =~ //Objects/.+\\.xml$/"
+				"when": "resource =~ /\\/Objects\\/.+\\.xml$/"
```
Comment (karl-anthony-ng) [inline packages/vscode-extension/package.json:142]: Are the changes here intentional?

Comment (davidecorreu) [reply packages/vscode-extension/package.json:142]: Those changes where in my workspace, I forgot to remove them.
They are the solution to not show the specific command in the context menu in any place as those `when` clauses are kind of broken.
I can leave them or remove them, as you wish. I did test them in my Windows machine and worked fine, but maybe is better to leave as it was.

Comment (karl-anthony-ng) [reply packages/vscode-extension/package.json:142]: I think we already have a separate issue to work on that part. Maybe better to move these changes to that issue?

Comment (davidecorreu) [reply packages/vscode-extension/package.json:142]: I'll remove those changes then. 👍

**Review by karl-anthony-ng — APPROVED** [review]

**Review by antoniofruci — APPROVED** [review]

**Pull request merged**